src/set.coffee

```diff
@@ -27,6 +27,7 @@ install = (_, jasmine) ->
   context = @
 
   globalPatches =
+    __autoIncrement: -1
```

Comment:
Now I wonder how did this ever work, if `__autoIncrement` was `undefined`, `__autoIncrement++` always returns `NaN` (and `NaN++` returns `NaN`). Weird :-S